build_ovtf.py

```diff
@@ -175,10 +175,11 @@ def main():
     build_dir = arguments.build_dir
 
     if (arguments.openvino_version not in [
-            "master", "2022.1.0", "releases/2022/2"
+            "master", "2022.1.0", "2022.2.0", "releases/2022/2"
     ]):
         raise AssertionError(
-            "Only 2022.1.0 OpenVINO version and master branch are supported")
+            "Only 2022.1.0, 2022.2.0, and master branch of OpenVINO are supported"
+        )
 
```

Comment:
Should we update it to 2022.2.0 tag below as well?

`    parser.add_argument(
        '--openvino_version',
        help="Openvino version to be used for building from source",
        default='releases/2022/2')
`

Reply:
Yes, we should change this once the OpenVINO final release tag is available.